docs/user-guide/ui/initial-configurations/first-steps.md

```diff
@@ -33,7 +33,7 @@ After **Log In** for the first time into your DAppNode you will see this screen.
    - Turn on the Use remote during syncing or errors
    - The client, we don't have a preference between geth, nethermind ...
 
-This configuration can be modified whatever you want. You can find all the information with all the details about this decision in [this section](./first-steps).
+This configuration can be modified whatever you want. You can find all the information with all the details about this decision in [this section](https://docs.dappnode.io/user-guide/ui/initial-configurations/select-a-client).
```

Comment:
@tropicar This is a very bad idea since now you can only ever render the page in this specific provider. If we ever change the domain everything will break